src/screens/Inventory/components/InventoryCard.tsx

```diff
@@ -41,9 +44,9 @@ function InventoryCard(props: InventoryCardProps) {
       <CardContent className={classes.cardContent}>
         <div className={classes.leftContent}>
           <Typography  variant="h2">{product.name}</Typography>
-          <Typography variant="body1" color="textSecondary">{`Last Updated: ${lastUpdated}`} </Typography>
+          <Typography variant="body1" color="textSecondary">{`${intl(words.last_updated_amount)}: ${lastUpdated}`} </Typography>
         </div>
-        <Typography variant="h2">{`${currentQuantity} ${product.unit}(s)`}</Typography>
+        <Typography variant="h2">{`${currentQuantity} ${product.unit}(${intl(words.s)})`}</Typography>
```

Comment:
Just checking: we're only translating the hardcoded strings, and the dynamic units / names / etc are meant to be in the proper language already right?

Reply:
Yup. I translate the letter `s` because it's meant to indicate plurality and it seems to be hardcoded everywhere throughout the app. Are you suggesting I should leave it in English?

Reply:
No, just checking! This is fine (we can have the NPO address it if desired behavior is different).